packages/jest-haste-map/src/crawlers/node.js

```diff
@@ -0,0 +1,121 @@
+ /**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+const H = require('../constants');
+
+const fs = require('fs');
+const path = require('path');
+const spawn = require('child_process').spawn;
+
+function find(roots, extensions, ignore, callback) {
+  const result = [];
+  let activeCalls = 0;
```

Comment:
I agree with you. I lifted this from node-haste1 and only updated the code style to fit ES2015 better. I figured I'll leave it as is so I don't break it :)